tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/LanguageServices/WorkspaceProjectContextProviderTests.cs

```diff
@@ -220,11 +220,10 @@ private static WorkspaceProjectContextProvider CreateInstance(UnconfiguredProjec
 
             var projectFaultService = IProjectFaultHandlerServiceFactory.Create();
             project ??= UnconfiguredProjectFactory.Create();
-            threadingService ??= IProjectThreadingServiceFactory.Create();
```

Comment:
Also remove the `threadingService` parameter from this method as it is now unused.